bin/fin

```diff
@@ -6292,6 +6293,15 @@ case "$1" in
 			_exec wp "$@"
 		fi
 		;;
+	composer)
+		load_configuration
+		shift
+		if [[ "$1" == "" ]]; then
+			_exec composer
```

Comment:
I guess there is no need in this for composer. Just do `_exec composer "$@"` all the time.
It was needed for drush for drush-specific reasons